dev/com.ibm.ws.security.jwtsso.token/resources/com/ibm/ws/security/jwtsso/token/resources/JWTSSOTOKENMessages.nlsprops

```diff
@@ -0,0 +1,26 @@
+###############################################################################
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME com.ibm.ws.security/resources/com/ibm/ws/security/jwtsso/token/resources/JWTSSOTOKENMessages.nlsprops
+#COMPONENTPREFIX CWWKS
+#COMPONENTNAMEFOR WebSphere Application Server JWTSSOTOKEN
+#ISMESSAGEFILE TRUE
+#NLS_MESSAGEFORMAT_VAR
+#NLS_ENCODING=UNICODE
+# -------------------------------------------------------------------------------------------------
+# Message prefix block: CWWKS6351 - CWWKS6399
+
+JWTSSO_CONFIG_INVALID=CWWKS6351E: The JWTSSO feature configuration is invalid. Check the JWTSSO feature configuration. Check for prior error messages in the log.
+JWTSSO_CONFIG_INVALID.explanation=A valid JWTSSO feature configuration could not be obtained.
+JWTSSO_CONFIG_INVALID.useraction=Check the JWTSSO feature configuration. Check for prior error messages in the log.
+
+JWTSSO_CONFIG_INVALID_OR_TOKEN_INVALID=CWWKS6352E: JWTSSO token validation failed. Check the JWTSSO feature configuration. Check for prior error messages in the log.
+JWTSSO_CONFIG_INVALID_OR_TOKEN_INVALID.explanation=Validation of the supplied JSON web token failed. The token or feature configuration might be invalid.
```

Comment:
"failed"-->"did not succeed"
"might be invalid"-->"might not be valid"

Reply:
argh, in your prior comment you said, Would it be better to say, "...token failed."?  I'll change it back.

Reply:
Sorry about this! A coworker clarified for me that we are not supposed to use "negative" words.